.travis/debian8/Dockerfile

```diff
@@ -11,15 +11,19 @@ RUN sed -i '/jessie-updates/d' /etc/apt/sources.list  # Now archived
 RUN apt-get update && apt-get install -y \
   apt-utils                              \
   build-essential                        \
-  cmake                                  \
   curl                                   \
   git                                    \
   lsb-release                            \
   omniidl                                \
   libomniorb4-dev                        \
   libcos4-dev                            \
   libomnithread3-dev                     \
-  libzmq3-dev
+  libzmq3-dev                            \
+  wget
+
+RUN wget --no-check-certificate https://cmake.org/files/v3.10/cmake-3.10.0-Linux-x86_64.sh -O /tmp/cmake-install.sh \
```

Comment:
Maybe we should install CMake 3.7 instead of 3.10 on Travis Debian 8 to be sure that it works fine with CMake 3.7?

Reply:
I orignally though about that as well but debian 9 has cmake 3.7.2, see https://packages.debian.org/stretch/cmake, which is close enough to 3.7 I think.

Reply:
> 
> 
> I orignally though about that as well but debian 9 has cmake 3.7.2, see https://packages.debian.org/stretch/cmake, which is close enough to 3.7 I think.

Fine by me.